src/SamplesApp/SamplesApp.Shared/App.xaml.cs

```diff
@@ -163,17 +163,18 @@ static void ConfigureFilters(ILoggerFactory factory)
 						// { "Windows.UI.Xaml.Controls.PopupPanel", LogLevel.Debug },
 
 						// Generic Xaml events
-						//{ "Windows.UI.Xaml", LogLevel.Debug },
+						// { "Windows.UI.Xaml", LogLevel.Debug },
 						// { "Windows.UI.Xaml.Shapes", LogLevel.Debug },
-						//{ "Windows.UI.Xaml.VisualStateGroup", LogLevel.Debug },
-						//{ "Windows.UI.Xaml.StateTriggerBase", LogLevel.Debug },
+						// { "Windows.UI.Xaml.VisualStateGroup", LogLevel.Debug },
+						// { "Windows.UI.Xaml.StateTriggerBase", LogLevel.Debug },
 						// { "Windows.UI.Xaml.UIElement", LogLevel.Debug },
+						{ "Windows.UI.Xaml.FrameworkElement", LogLevel.Trace },
```

Comment:
Remove

Reply:
It's a change in a commit named "WIP" ;-)